datastore/tokenstore/interface.go

```diff
@@ -17,7 +17,7 @@ var (
 )
 
 const (
-	serviceName = "googlephotos-uploader-go-api"
+	serviceName = "googlephotosuploadergoapi"
```

Comment:
Hi @pdecat 

Have you found where are special characters defined? I've been looking around without any success!

We can use `gPhotosUploader` instead... right?

Reply:
Replaced `googlephotosuploadergoapi` by `gPhotosUploader`.

Reply:
Did a bit of additional debugging and the encoding happens D-Bus server side.
The first time the encoded value is visible client side is here: https://github.com/gsterjov/go-libsecret/blob/a6f4afe/service.go#L91
Maybe this could be stored somewhere.

Then found D-Bus object path spec:
> Each element must only contain the ASCII characters "[A-Z][a-z][0-9]_"
 https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-marshaling-object-path